code/drasil-code/Language/Drasil/Code/Imperative/Doxygen/Import.hs

```diff
@@ -765,7 +765,7 @@ makeDoxConfig prog opt p =
   text "# messages are off.",
   text "# The default value is: NO.",
   blank,
-  text "QUIET                  = NO",
+  text "QUIET                  = YES",
```

Comment:
@JacquesCarette @smiths Is this something we think users might care enough about that we should make it toggle-able with a `Choice`? Would be easy to do.

Reply:
My vote is Yes - we should make this choice toggle-able.  I think it is the right thing to do, and it gives us another example of user customization, and it is easy to do.  :-)

Reply:
But should be done 'later', so that an issue should be filed rather than doing it right now.

Reply:
#1871 created.